pallets/ocex/src/benchmarking.rs

```diff
@@ -400,26 +376,6 @@ benchmarks! {
 	set_snapshot {
 		let call = Call::<T>::set_snapshot{ new_snapshot_id: u64::MAX };
 	}: { call.dispatch_bypass_filter(RawOrigin::Root.into())? }
-
-	change_pending_withdrawal_limit {
-		let origin = T::GovernanceOrigin::try_successful_origin().unwrap();
-		let call = Call::<T>::change_pending_withdrawal_limit { new_pending_withdrawals_limit: u64::MAX };
-	}: { call.dispatch_bypass_filter(origin)? }
```

Comment:
Trait [member still present](https://github.com/Polkadex-Substrate/Polkadex/blob/843169169b0400c279d380eefbbe0437955faa8d/pallets/ocex/src/lib.rs#L123).

Reply:
Removed it